tests/DataFixtures/ORM/PromotionsFixture.php

```diff
@@ -22,57 +24,102 @@ public function getDependencies()
 
     public function load(ObjectManager $manager)
     {
+        /*
+        PROMOTION CONTEXTS:
+        ===================
+
+        Brand (b010t19z)
+            promotion 0 (p000000h)
+
+            series 1 (b00swyx1)
+                promotion 1 (p000001h - SUPER PROMOTION)
+                promotion 2 (EXPIRED)
+
+                episode 1
+                    promotion 4 (p000004h)
+
+            series 2 (b010t150)
+                promotion 3 (EXPIRED)
+                image (mg000003)
+
+        */
+
         $this->manager = $manager;
 
         $this->buildPromotion(
             'p000000h',
-            'active promotion of CoreEntity',
-            $this->getReference('b00swyx1'), // promotion of Core entity - series 1
-            1, // weight
+            'promotion 1',
+            $this->getReference('b0175lqm'), // promoting: series2/episode1
+            5,
             true,
             new DateTime('1900-01-01 00:00:00'),
-            new DateTime('3000-01-01 00:00:00')
+            new DateTime('3000-01-01 00:00:00'),
```

Comment:
[Thanks to PHP having weirdly niche but occasionally neat and expressive stuff in it](http://php.net/manual/en/datetime.formats.relative.php) you could do `new DateTime('-1 year')` and `new DateTime('+1 year')` to generate relative timestamps?

Not convinced that's more readable than fixed far-future things though :)